src/Microsoft.AspNetCore.Routing/Patterns/RoutePattern.cs

```diff
@@ -48,10 +48,10 @@ public sealed class RoutePattern
         public IReadOnlyDictionary<string, object> Defaults { get; }
 
         /// <summary>
-        /// Gets the set of constraint references for the route pattern.
-        /// The keys of <see cref="Constraints"/> are the route parameter names.
+        /// Gets the set of parameter policy references for the route pattern.
+        /// The keys of <see cref="ParameterPolicies"/> are the route parameter names.
         /// </summary>
-        public IReadOnlyDictionary<string, IReadOnlyList<RoutePatternConstraintReference>> Constraints { get; }
+        public IReadOnlyDictionary<string, IReadOnlyList<RoutePatternParameterPolicyReference>> ParameterPolicies { get; }
```

Comment:
What should happen here? Should all reference to constraints on `RoutePattern` and `RoutePatternFactory` be replaced with parameter policy?